bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/i18n/gree_DE.properties

```diff
@@ -4,90 +4,89 @@ binding.gree.label = GREE Air Conditioner
 binding.gree.description = Dieses Binding integriert Klimaanlagen der Marke GREE
 
 # thing types
-thing-type.gree.airconditioner.label = GREE Klimaanlage
+thing-type.gree.airconditioner.label = GREE 
 thing-type.gree.airconditioner.description = Eine GREE Klimaanlage mit WiFi Modul
 
 # thing type config description
 thing-type.config.gree.airconditioner.ipAddress.label = IP Adresse
-thing-type.config.gree.airconditioner.ipAddress.description = IP Adresse des GREE-Ger�tes.
+thing-type.config.gree.airconditioner.ipAddress.description = IP Adresse des GREE-Gerätes.
```

Comment:
This file should have been resetted to the old one (with correct encoding) and then changes applied again. Now all the Umlaut characters show up as changes :-(

Reply:
@tobiasbayer I'm done with the review, please apply the requested changes and make sure that the encoding of the property files are UTF-8

Reply:
The properties files should be encoded `ISO-8859` as per definition of java properties files and not `UTF-8`.

Reply:
It's my fault, that the file is encoded in UTF-8. I wasn't aware that the spec states ISO-8859. [Only from JDK 9](https://docs.oracle.com/javase/9/intl/internationalization-enhancements-jdk-9.htm#JSINT-GUID-974CF488-23E8-4963-A322-82006A7A14C7) on it's UTF-8.

Reply:
should be fixed by restoring file encoding, correct?